README.md

```diff
@@ -35,27 +70,51 @@ This guide will help you set up Volto-Hydra (AdminUI) and connect your frontend
     make start
     ```
 
-### Step 2: Setting Up Your Frontend
+### Make your frontend editable
 
-**1. Choose Your Framework**
+- Take the latest [hydra.js](https://github.com/collective/volto-hydra/tree/hydra.js) and include it in your frontend
```

Comment:
added hydra.js branch consisting `hydra.js` file and we should add detailed instruction on its readme about its integration and features.